olive/olive_config.json

```diff
@@ -95,10 +95,10 @@
             "supported_precisions": [ "fp16" ]
         },
         "OnnxIOFloat16ToFloat32": {
```

Comment:
Why is the key not changed to match the pass name?

Reply:
Mb, updated!